src/decode.c

```diff
@@ -781,6 +801,22 @@ void DecodeGlobalConfig(void)
             decoder_max_layers = value;
         }
     }
+    GetPacketMaxAlertConfig();
+}
+
+void GetPacketMaxAlertConfig(void)
+{
+    intmax_t max = 0;
+    if (ConfGetInt("packet-alert-max", &max) == 1) {
+        if (max < 0 || max > UINT8_MAX) {
```

Comment:
Q: What happens in a case when max == 0?

Reply:
I think we decided that if the user chose that, that was up to them... 🤔

Reply:
change behavior to Warning and default